### PR TITLE
Create international_keymap.js

### PR DESCRIPTION
**Pull request by AKuederle**
Expanded the toggle-command extension to include the indent and dedent commands with keybindings, which will work better on non-US-keyboards.

I'm unsure about the name, and if further steps are required to make this fit in this repo.

### CODE REVIEW

File: usability/international_keymap.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    load_ipython_extension: function (){
+        IPython.keyboard_manager.actions.register(toggle_comments,'toggle_comments');
+        IPython.keyboard_manager.actions.register(indent_selection,'indent_selection');
+        IPython.keyboard_manager.actions.register(dedent_selection,'dedent_selection');
+        IPython.keyboard_manager.edit_shortcuts.add_shortcut('Alt-3', 'auto.toggle_comments');
+        IPython.keyboard_manager.edit_shortcuts.add_shortcut('Alt-2', 'auto.dedent_selection');
+        IPython.keyboard_manager.edit_shortcuts.add_shortcut('Alt-1', 'auto.indent_selection');
+        IPython.keyboard_manager.command_shortcuts.add_shortcut('Shift-k','ipython.move-selected-cell-up');
+        IPython.keyboard_manager.command_shortcuts.add_shortcut('Shift-j','ipython.move-selected-cell-down');
+        console.log("I have been loaded ! -- interantional_shortcuts");
```
Comment (Carreau) [inline usability/international_keymap.js]: inter**an**tional

**Comment by juhasch** [comment]
Is `Tab` / `Shift-Tab` not working as indent/dedent operation for you ?

**Comment by AKuederle** [comment]
Of course it does,  but ''tab'' inserts the tab at the cursor position and not at the beginning of the line.  The indent and dedent functions here will work independent of the cursor's position (like ctrl+[ on international keyboards)

**Comment by juhasch** [comment]
OK, I understand. Two questions:
 * Do you want the help_index always to be `zz` ?
 * Can you add a short description in the Wiki what this extension does ?
Otherwise I am fine merging.

**Comment by AKuederle** [comment]
I personally think its better to have the thing you added through extensions/ modifications at the end of the listing. However, its no big deal to change it.
And, of course, I will gladly add a description in the wiki. Should I wait until the PR is actual merged before I do so?

**Comment by juhasch** [comment]
Sorry for the delay. Please go ahead and add a description to the Wiki

**Comment by AKuederle** [comment]
Wrote the wiki article. If there is anything I should add, just say so!
Just for the record, I tested the extension with the current Jupyter version and everything works fine. So, maybe the extension can also be moved to the master-branch in the future.

**Pull request merged**